ctags/parsers/c.c

```diff
@@ -1221,11 +1219,10 @@ static void addOtherFields (tagEntryInfo* const tag, const tagType type,
 			{
 				tag->extensionFields.access = accessField (st);
 			}
-            if ((true == st->gotArgs) && (true == Option.extensionFields.argList) &&
```

Comment:
At least Option.extensionFields.argList doesn't exist. Since it was always TRUE for us, I didn't investigate more.